ext/mixed/js/display-generator.js

```diff
@@ -258,6 +276,117 @@ class DisplayGenerator {
         return node;
     }
 
+    createPitches(details) {
+        if (!this._termPitchAccentStaticTemplateIsSetup) {
+            this._termPitchAccentStaticTemplateIsSetup = true;
+            const t = this._templateHandler.instantiate('term-pitch-accent-static');
+            document.head.appendChild(t);
+        }
+
+        const [dictionary, dictionaryPitches] = details;
+
+        const node = this._templateHandler.instantiate('term-pitch-accent-group');
+        node.dataset.dictionary = dictionary;
+        node.dataset.pitchesMulti = 'true';
+        node.dataset.pitchesCount = `${dictionaryPitches.length}`;
+
+        const tag = this.createTag({notes: '', name: dictionary, category: 'dictionary'});
+        node.querySelector('.term-pitch-accent-group-tag-list').appendChild(tag);
+
+        const n = node.querySelector('.term-pitch-accent-list');
+        DisplayGenerator._appendMultiple(n, this.createPitch.bind(this), dictionaryPitches);
+
+        return node;
+    }
+
+    createPitch(details) {
+        const {expressions, reading, position, tags} = details;
+        const morae = DisplayGenerator._jpGetKanaMorae(reading);
+
+        const node = this._templateHandler.instantiate('term-pitch-accent');
+
+        node.dataset.pitchAccentPosition = `${position}`;
+        node.dataset.tagCount = `${tags.length}`;
+
+        let n = node.querySelector('.term-pitch-accent-position');
+        n.textContent = `${position}`;
+
+        n = node.querySelector('.term-pitch-accent-tag-list');
+        DisplayGenerator._appendMultiple(n, this.createTag.bind(this), tags);
+
+        n = node.querySelector('.term-pitch-accent-expression-list');
+        DisplayGenerator._appendMultiple(n, this.createPitchExpression.bind(this), expressions);
```

Comment:
Instead of `expressions`, I think this could have something similar to `only` here:
https://github.com/FooSoft/yomichan/blob/46fee07d36b3966af9bacca8c6253b044bde07ee/ext/bg/js/dictionary.js#L282-L293
Now there are issues when either multiple expressions have the same pitch accent:
![2020-03-03_00-26-34](https://user-images.githubusercontent.com/5602483/75724473-22d7f300-5ce7-11ea-93e7-c6cbc4d690fc.png)
Or only graphs are enabled:
![2020-03-03_00-27-34](https://user-images.githubusercontent.com/5602483/75724466-1f446c00-5ce7-11ea-879b-7f3aed049866.png)

Reply:
> Instead of `expressions`, I think this could have something similar to `only` 

That's effectively already what it's doing

> [img1]

Is the issue here that `背、脊：` is displayed twice?

> [img2]

Is the issue here that there is no reading displayed?

Reply:
The problem is that it only works when all expressions differ in both reading and kanji. In the 背 example the pitch expression pointers are not needed because both pitch accents apply to both results, and in the 端 example the logic thinks that everything is fine although there are three different readings.

Reply:
I'll look into making this implementation more consistent with the `only` implementation, but there are potentially ambiguous results with that method as well:

端/はた: Which kanji does the (はた only) apply to?
![image](https://user-images.githubusercontent.com/11037431/75942332-1aef8e80-5e60-11ea-94a6-a26b490793ea.png)

Reply:
If you mean the JMdict's definition "third person", it applies to every kanji. The kanji is left out to keep it brief. It tries to emulate the original JMdict format which looks like this (from [my abandoned project](https://github.com/siikamiika/mecab-translate)):
![2020-03-05_14-15-10](https://user-images.githubusercontent.com/5602483/75980749-c691f080-5eeb-11ea-9b11-9015fe765c36.png)

I've also been thinking of a display format where the readings are grouped under each kanji, but that could be problematic with cross browser ruby text and Anki. Better to fix the data structure first anyway https://github.com/FooSoft/yomichan/issues/337.